source/puppet/4.2/reference/release_notes.markdown

```diff
@@ -20,19 +17,91 @@ Make sure you also read the [Puppet 4.0 release notes](/puppet/4.0/reference/rel
 
 Also of interest: the [Puppet 4.1 release notes](/puppet/4.1/reference/release_notes.html).
 
+## Puppet 4.2.3
+
+Released October 29, 2015
+
+Shipped in puppet-agent version 1.2.7.
+
+Puppet 4.2.3 is a bug fix and platform support release in the Puppet 4.2 series. It also removes support for Ubuntu 14.10 (Utopic).
+
+For JIRA issues related to Puppet 4.2.3, see:
+
+* [Fixes for Puppet 4.2.3]()
+* [Introduced in Puppet 4.2.3]()
+
+### New Platforms: AIX 5.3, 6.1, and 7.1, and Solaris 11
+
+Puppet 4.2.3 supports AIX 5.3, 6.1, and 7.1 on PowerPC architectures, and Solaris 11 on SPARC and i386 architectures.
```

Comment:
it sounds like puppet only supports Solaris 11, but we added support for 10 in the last release. Could we say 4.2.3 "adds support for..."